src/main.ts

```diff
@@ -50,9 +52,13 @@ async function bootstrap(): Promise<void> {
     const document = SwaggerModule.createDocument(app, options);
     SwaggerModule.setup('/', app, document);
 
-    // await app.startAllMicroservicesAsync();
+    // Connects to Rabbit etc
+    await app.startAllMicroservicesAsync();
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     const port = config.get('http.port') ?? 3000;
     await app.listen(port, '0.0.0.0');
+    // eslint-disable-next-line no-console
     console.log(`Application is running on: ${await app.getUrl()}`);
```

Comment:
Probably want to change this to a logger. https://docs.nestjs.com/techniques/logger

Reply:
Will be done as part of #26